js/glotdict-validation.js

```diff
@@ -62,7 +62,12 @@ function gd_search_glossary_on_translation(e, selector) {
       });
       if (reset !== '') {
         howmany++;
-        jQuery('.textareas', $editor).prepend(gd_get_warning('The translation is missing the suggested translation ' + reset + 'for "<i>' + term + '</i>"', discard));
+        reset = reset.slice(0, -2);
```

Comment:
can you explain that part? this will remove part of a string but why?

Reply:
@Mte90 I thought that myself initially till I dove in and seems we always append the comma and space;
https://github.com/Mte90/GlotDict/blob/master/js/glotdict-validation.js#L57

Which is what the slice is for.

Reply:
uh right is only a cleaning of what we are doing!

Reply:
that little cleanup is probably easier than checking if you are on the last index and avoiding the additional comma and space :)